debian/changelog

```diff
@@ -5,6 +5,11 @@ daos (1.3.0-9) unstable; urgency=medium
  -- Mohamad Chaarawi <mohamad.chaarawi@intel.com>  Wed, 16 Apr 2021 17:57:00 -0400
 
 daos (1.3.0-8) unstable; urgency=medium
+  [ Steven Bollinger ]
+  * Create cart_utils library
+
+ -- Steven A. Bollinger<stevenx.bollinger@intel.com>  Tue, 15 Apr 2021 13:30:00 -0400
+
```

Comment:
this looks like merge gone bad, it should be its own entry for version 1.3.0-10 at the top

Reply:
How/what do you mean.  It looks like its own entry.  Where did you get the number 1.3.0-10

Reply:
Fixed